### PR TITLE
Pods Support + Classic/Pods Hybrid Support

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
pods structure can now be migrated from.

This PR just adds to the existing code.
I think a future PR should be a refactor. Lots of things need to be extracted to methods for easier inheritance.

### CODE REVIEW

File: test/fixtures/pods-acceptance/output.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+module.exports = {
```
Comment (NullVoxPopuli) [inline test/fixtures/pods-acceptance/output.js]: output should be the same as the classic acceptance input

Comment (rwjblue) [inline test/fixtures/pods-acceptance/output.js]: You could require that and return it if you wanted (just to ensure both stay in sync)...

Comment (NullVoxPopuli) [inline test/fixtures/pods-acceptance/output.js]: I like that idea, thanks!

**Comment by rwjblue** [comment]
This is looking really good! Thank you so much for working on it!

@mixonic / @andyhot - Mind reviewing also?

**Comment by Subtletree** [comment]
Looks like this currently wouldn't work for apps with no `podModulePrefix` or for any with `podModulePrefix` !== 'pods'. Is that right?

**Comment by andyhot** [comment]
@rwjblue taking a look this weekend
@Subtletree unless there's a regression (which judging from the passing tests there shouldn't) classic structure just works. If there's indeed a requirement for podModulePrefix to be 'pods', we'll fix that - i too have apps where that's not the case :)

**Comment by NullVoxPopuli** [comment]
how do you get access to the `podModulePrefix`?

**Comment by NullVoxPopuli** [comment]
so, here is my plan:
I'm going to assume `config/environment.js`, and require/import that file, and get the podModulePrefix that way.

Is there any scenario where that wouldn't work?

File: bin/ember-module-migrator [hunk 2]
```diff
@@ -8,7 +8,9 @@
 // Provide a title to the process in `ps`
 process.title = 'ember-module-migrator';
 
-var Migrator = require('../lib');
+var fs = require('fs');
+
+var Migrator = require('../lib');i
```
Comment (josemarluedke) [inline bin/ember-module-migrator]: It seems to have a type here `;i`

Comment (NullVoxPopuli) [reply bin/ember-module-migrator]: shoot!

File: bin/ember-module-migrator [hunk 3]
```diff
@@ -20,10 +22,15 @@ var parsed = nopt(options);
 
 var projectRoot = parsed['project-root'] || process.cwd();
 var projectName = parsed['project-name'];
+
+var environment = require(projectRoot + '/config/environment.js');
+
+var modulePrefix = environment.modulePrefix;
+var podModulePrefix = environment.podModulePrefix.replace(modulePrefix + '\/', '');
```
Comment (NullVoxPopuli) [inline bin/ember-module-migrator]: based on my app:

```js
    modulePrefix: 'aeonvera',
    podModulePrefix: 'aeonvera/pods',
```

**Comment by NullVoxPopuli** [comment]
Work on namespaceless pods: https://github.com/NullVoxPopuli/ember-module-migrator/pull/1

**Comment by NullVoxPopuli** [comment]
namespacelessness is gettitng there. I have a path to doing it, just need to work out some kinks

**Comment by NullVoxPopuli** [comment]
Node 4 had a syntax error????? `SyntaxError: Unexpected token {`

**Comment by NullVoxPopuli** [comment]
The failing tests are 'just' timeouts. :-\

**Comment by NullVoxPopuli** [comment]
@rwjblue squashed!

**Comment by NullVoxPopuli** [comment]
eslint check on node 7 timed out

**Comment by rwjblue** [comment]
restarted that one, but we should drop Node 7 in a followup PR

**Pull request merged**